Exercise_D and E/index.html

```diff
@@ -9,13 +9,44 @@
     <link rel="stylesheet" href="./assets/style/index.css" />
     <link
       rel="stylesheet"
+      <!--
       href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
-      integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
-      crossorigin="anonymous"
+      --
     />
   </head>
   <body>
     <!-- CODE HERE -->
+    <nav>
+      <img src="assets/images/logo.png" alt="" />
+    </nav>
+
+    <section class="tophead">
+      <div class="overlay">
+        <div class="wrapper">
+          <h1 class="logo">Beautiful Flowers</h1>
+          <button type="button" class="learnbut">Learn More</button>
+        </div>
+      </div>
+    </section>
+
+    <section id="bottomicons">
+      <div class="littlebut">
+        <img src="assets/images/flower-icon.png" />
+        <p>Flowers for every occassion</p>
+      </div>
+      <div class="littlebut">
+        <img src="assets/images/payment-icon.png" />
+        <p>All major payment types accepted</p>
+      </div>
+      <div class="littlebut">
+        <img src="assets/images/phone-icon.png" />
+        <p>Free Consultation</p>
+      </div>
+      <div class="littlebut">
+        <img src="assets/images/truck-icon.png" />
+        <p>City wide delivery</p>
+      </div>
+    </section>
 
```

Comment:
Well done Mark I can see the effort that you have put in to complete these exercises!